### ISSUE TITLE
is it possible to add custom columns in the task list table?

### ISSUE THREAD

**Issue by MrWorldWide101** [MaTeMaTuK/gantt-task-react#58]

**Comment by Graeme43** [comment]
Yes, by overriding providing the default TaskListTable and TaskListHeader functions. You will need to extend the Task interface if you want to add new columns in the data that you provide. See the styling options in the documentation at the bottom [Link](https://github.com/MaTeMaTuK/gantt-task-react#stylingoption)

**Comment by Ankit10Kumar** [comment]
can you give an example code , because I didn't understand what you said above.

**Comment by ar7max** [comment]
> can you give an example code , because I didn't understand what you said above.

take a look at these two files as references:

[task-list-table.tsx](https://github.com/MaTeMaTuK/gantt-task-react/blob/main/src/components/task-list/task-list-table.tsx)

[task-list-header.tsx](https://github.com/MaTeMaTuK/gantt-task-react/blob/main/src/components/task-list/task-list-header.tsx)

You can implement ur own components and render whatever you want. Just provide ur custom components as TaskListHeader and TaskListTable according to GanttProps interface [public-types.ts](https://github.com/MaTeMaTuK/gantt-task-react/blob/main/src/types/public-types.ts#L121)
